Dockerfile

```diff
@@ -0,0 +1,56 @@
+# Base node stage that sets up common config for dev & prod
+FROM node:12-alpine as node
+WORKDIR /opt/cord-api
+
+# Add wait-for utility to make waiting for db easier (see usage in docker-compose.yml)
+# From https://github.com/eficode/wait-for/pull/9 (has fixes for env handling)
+ADD https://raw.githubusercontent.com/eficode/wait-for/96511d65c6578d4866591283fdec6e2fba7e6770/wait-for /usr/local/bin/wait-for
+RUN chmod +x /usr/local/bin/wait-for
+
+ENV NODE_ENV=development \
+    NEO4J_URL= \
+    NEO4J_USERNAME= \
+    NEO4J_PASSWORD= \
+    JWT_AUTH_KEY= \
+    FILES_S3_BUCKET= \
+    PORT=80
```

Comment:
You can drop this line since we're exposing port `80` raw in the line below

Reply:
App still needs to be configured on that port 😉

Reply:
Touche!